ext/libgav1_android.sh

```diff
@@ -0,0 +1,41 @@
+# If you want to use a local build of libgav1 to be used with the android NDK,
+# you must clone the libgav1 repo in this directory first, then enable CMake's
+# AVIF_CODEC_LIBGAV1 and AVIF_LOCAL_LIBGAV1 options.
+#
+# This script will build libgav1 for the default ABI targets supported by
+# android. You must pass the path to the android NDK as a parameter to this
+# script.
+#
+# Android NDK: https://developer.android.com/ndk/downloads
+#
+# The git SHA below is known to work, and will occasionally be updated. Feel
+# free to use a more recent commit.
+#
+# cmake and make must be in your PATH.
+
+if [ $# -ne 1 ]; then
+  echo "Usage: ${0} <path_to_android_ndk>"
+else
+  git clone -b v0.17.0 --depth 1 https://chromium.googlesource.com/codecs/libgav1
+
+  cd libgav1
+  git clone -b lts_2021_03_24 --depth 1 https://github.com/abseil/abseil-cpp.git third_party/abseil-cpp
+  mkdir build
+  cd build
+
+  ABI_LIST="armeabi-v7a arm64-v8a x86 x86_64"
+  for abi in ${ABI_LIST}; do
+    mkdir "${abi}"
+    cd "${abi}"
+    cmake ../.. \
+      -DCMAKE_TOOLCHAIN_FILE=../../cmake/toolchains/android.cmake \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DLIBGAV1_ANDROID_NDK_PATH=${1} \
+      -DLIBGAV1_THREADPOOL_USE_STD_MUTEX=1 \
+      -DANDROID_ABI=${abi}
+    make -j
```

Comment:
Nit: Is it possible to use ninja instead of make?

Reply:
Done.